docs/platform/one-click/deploying-mongodb-with-one-click-apps/index.md

```diff
@@ -0,0 +1,106 @@
+| **Linode Plan** | Your Linode's [hardware resources](/docs/platform/how-to-choose-a-linode-plan/#hardware-resource-definitions). Since MongoDB can require a significant amount of RAM, we recommend using a [high memory Linode](https://www.linode.com/pricing/high-memory). You can always [resize your Linode](/docs/platform/disk-images/resizing-a-linode/) to a different plan later if you feel you don't need you need these resources are needed. *Required*. |
+| **Linode Label** | The name for your Linode, which must be unique between all of the Linodes on your account. This name will be how you identify your server in the Cloud Manager’s Dashboard. *Required*. |
+| **Root Password** | The primary administrative password for your Linode instance. This password must be provided when you log in to your Linode via SSH. It must be at least 6 characters long and contain characters from two of the following categories: lowercase and uppercase case letters, numbers, and punctuation characters. Your root password can be used to perform any action on your server, so make it long, complex, and unique. *Required*. |
+
+When you've provided all required Linode Options, click on the **Create** button. **Your MongoDB app will complete installation anywhere between 2-5 minutes after your Linode has finished provisioning**.
+
+## Getting Started after Deployment
+
+### Access your MongoDB Site
+
+After MongoDB has finished installing, you will be able to access MongoDB from the console via ssh with your Linode's IPv4 address:
+
+1.  [SSH into your Linode](/docs/getting-started/#connect-to-your-linode-via-ssh) and [create a limited user account](/docs/security/securing-your-server/#add-a-limited-user-account).
+
+1.  Log out and log back in as your limited user account.
+
+1.  Update your server:
+
+        sudo apt-get update && apt-get upgrade
+
+1.  Access MongoDB with the admin account password you set when launching the One-Click App:
+
+        mongo -u admin -p --authenticationDatabase admin
+
+    The `-u`, `-p`, and `--authenticationDatabase` options in the above command are required in order to authenticate connections to the shell. Without authentication, the MongoDB shell can be accessed but will not allow connections to databases.
+
+    The `admin` user purely administrative based on the roles specified. It is defined as an administrator of user for all databases, but does not have any database permissions itself. You may use it to create additional users and define their roles. If you are using multiple applications with MongoDB, set up different users with custom permissions for their corresponding databases.
```

Comment:
I think this paragraph is missing an 'is' in between "user" and "purely." Also, is "defined as an administrator of user**s**," I think?

Reply:
fixed